example/public/html/secured.html

```diff
@@ -23,11 +24,19 @@
 <main class="main">
     <div class="content">
         <div class="profile">
-            <div class="profile__title">
+            <div>
                 <h1>My Profile</h1>
             </div>
-            <div class="profile__email">
-                <span>{{ .user }}</span>
+            <div>
+                <span id="email"></span>
+            </div>
+            <div class="placeholder"></div>
+            <div class="passkeys">
+                <h2>Passkeys</h2>
+                <div>
+                    <div>Create a new passkey to sign in without a password.</div>
```

Comment:
Let's say "Create a new passkey on this device or on another device." to avoid using the word 'password' here and also hint to the caBLE use case.